update existed tutorial for signer (#351)
paritytech/wiki: Parity-Signer-Mobile-App-Apps-Kusama-tutorial.md

- [iOS](https://itunes.apple.com/us/app/parity-signer/id1218174838)

## 2. Setup or recover an Identity
When launching the app for the first time, no identity has been set up yet. At this stage, you will either want to create an identity directly from your mobile device or recover an identity previously created.
 
### Create an identity
 

> Tap on the `Create` button after you first entered into the app, and then you can give the identity any name you like.

Reviewer: ```suggestion
Tap on the `Create` button, and give a name to this identity.
```

In the next step, your recovery phrase will be presented to you. Think of it as a master key. If you lose it, you lose your money.
**Write this recovery phrase down and store it in a safe place**.
If your phone gets stolen/broken/forgotten this will be the only way to [recover your account](#recover-an-account-with-your-recovery-phrase).

You will then be asked to choose a pin code. This pin will be needed later on to unlock your account to manage the identity or sign a transaction.

> The next screen you will allow you to select a network for generating an account.

Reviewer: ```suggestion
The next screen will allow you to select a network to generate an account.
```

> If you choose Ethereum networks, the related Ethereum account will be generated for the account,

Reviewer: ```suggestion
If you choose an Ethereum network, the related Ethereum account will be generated for the identity,
```

> If you choose Substrate networks like Kusama, you will first create a root account, and then you will be able to derive more accounts with specified paths and names. The name could be changed later on, but once the path is set, it can not be changed. More information about path derivation see [here](Parity-Signer-Hierarchical-Deterministic-Key-Derivation).

Reviewer: ```suggestion
If you choose a Substrate network (like Kusama), you will first create a root account, and then you will be able to derive more accounts with specified paths and names. The name can be changed later on, but once the path is set, it can not be changed. More information about path derivation see [here](Parity-Signer-Hierarchical-Deterministic-Key-Derivation).
```

> For each derived account, you will be able to see the address and its related QR codes.

Reviewer: ```suggestion
For each derived account, you will be able to see the address and its related QR code.
```

![create account](images/Parity-Signer-apps-0.png)

### Recover an identity with your recovery phrase

If you already have an account created with either Parity Signer or any other waller, you can recover it by doing so:
- Tap on the top right side user icon, and choose ` + Add Identity`.
- Input the new identity name and tap the `Recover Identity` button.
- Type in the recovery phrase, word suggestion helps you prevent any typo. The field will turn red if the recovery phrase is not a bip39.
- Tap `Recover Identity`.
- Select a PIN number and confirm it by typing it again.
- Identity generated, now you can select the network to create the first account.

NOTICE: For V3 user, after recovering the seed phrase of Kusama account, the account will appear as an identity root account aside with identity name in the network selection screen.
 

## 3. Add Parity Signer's account to Polkadot-js apps

> To be able to follow this tutorial and interact with the Blockchain from a freshly created account on Parity Signer, you will need to get some KSMs on this account first. Polkadot-js apps allow you to manage your Signer account seamlessly.

Reviewer: it's called `apps` but it's actually one website
```suggestion
To be able to follow this tutorial and interact with the Blockchain from a freshly created account on Parity Signer, you will need to get some KSMs on this account first. Polkadot-js apps allows you to manage your Signer account seamlessly.
```
